internal/services/appservice/helpers/function_app_schema.go

```diff
@@ -334,14 +334,323 @@ func SiteConfigSchemaLinuxFunctionApp() *pluginsdk.Schema {
+					Description:  "The Connection String for linking the Windows Function App to Application Insights.",
+				},
+
+				"application_stack": windowsFunctionAppStackSchema(),
+
+				"app_service_logs": FunctionAppAppServiceLogsSchema(),
+
+				"default_documents": {
+					Type:     pluginsdk.TypeList,
+					Optional: true,
+					Computed: true,
+					Elem: &pluginsdk.Schema{
+						Type: pluginsdk.TypeString,
+					},
+					Description: "Specifies a list of Default Documents for the Windows Web App.",
+				},
+
+				"elastic_instance_minimum": {
+					Type:        pluginsdk.TypeInt,
+					Optional:    true,
+					Computed:    true,
+					Description: "The number of minimum instances for this Windows Function App. Only affects apps on Elastic Premium plans.",
+				},
+
+				"http2_enabled": {
+					Type:        pluginsdk.TypeBool,
+					Optional:    true,
+					Default:     false,
+					Description: "Specifies if the http2 protocol should be enabled. Defaults to `false`.",
+				},
+
+				"ip_restriction": IpRestrictionSchema(),
+
+				"scm_use_main_ip_restriction": {
+					Type:        pluginsdk.TypeBool,
+					Optional:    true,
+					Default:     false,
+					Description: "Should the Windows Function App `ip_restriction` configuration be used for the SCM also.",
+				},
+
+				"scm_ip_restriction": IpRestrictionSchema(),
+
+				"load_balancing_mode": { // Supported on Function Apps?
+					Type:     pluginsdk.TypeString,
+					Optional: true,
+					Default:  "LeastRequests",
+					ValidateFunc: validation.StringInSlice([]string{
+						"LeastRequests", // Service default
+						"WeightedRoundRobin",
+						"LeastResponseTime",
+						"WeightedTotalTraffic",
+						"RequestHash",
+						"PerSiteRoundRobin",
+					}, false),
+					Description: "The Site load balancing mode. Possible values include: `WeightedRoundRobin`, `LeastRequests`, `LeastResponseTime`, `WeightedTotalTraffic`, `RequestHash`, `PerSiteRoundRobin`. Defaults to `LeastRequests` if omitted.",
+				},
+
+				"managed_pipeline_mode": {
+					Type:     pluginsdk.TypeString,
+					Optional: true,
+					Default:  string(web.ManagedPipelineModeIntegrated),
+					ValidateFunc: validation.StringInSlice([]string{
+						string(web.ManagedPipelineModeClassic),
+						string(web.ManagedPipelineModeIntegrated),
+					}, false),
+					Description: "The Managed Pipeline mode. Possible values include: `Integrated`, `Classic`. Defaults to `Integrated`.",
+				},
+
+				"pre_warmed_instance_count": {
+					Type:        pluginsdk.TypeInt,
+					Optional:    true,
+					Computed:    true, // Variable defaults depending on plan etc
+					Description: "The number of pre-warmed instances for this function app. Only affects apps on an Elastic Premium plan.",
+				},
+
+				"remote_debugging": {
+					Type:        pluginsdk.TypeBool,
+					Optional:    true,
+					Default:     false,
+					Description: "Should Remote Debugging be enabled. Defaults to `false`.",
+				},
+
+				"remote_debugging_version": {
```

Comment:
should we include what version this referse to?
```suggestion
				"remote_debugging_visualstudio_version": {
```

Reply:
The possible values include the prefix `VS` so I think including `visualstudio` in the property name may be confusing here, possibly leading to users expecting to use `2017`, `2019` etc instead?